redundant-bmc/src/meson.build

```diff
@@ -3,7 +3,10 @@ execinstalldir = join_paths(get_option('libexecdir'), 'phosphor-state-manager')
 
 ssl_dep = dependency('openssl')
 
+gpiodcxx = dependency('libgpiodcxx', fallback: ['libgpiod', 'gpiodcxx_dep'])
```

Comment:
Just a doubt: Why are we not using the DBus call?

https://libgpiod.readthedocs.io/en/latest/dbus.html

https://github.com/openbmc/openbmc/blob/master/meta-openembedded/meta-oe/recipes-support/libgpiod/libgpiod_2.2.bb#L60C60-L60C80

I couldn't find the GPIO manager service in the system, so I'm unsure whether DBus-based GPIO interaction is recommended or not in OpenBMC.

Reply:
We are still pinned back on an older version upstream, because the new version with D-Bus is incompatible because it removed the find_line function which is used all over the place.  (Because technically there could be the same line name across multiple GPIO chips and they didn't like that).  I think it best if we wait for upstream to come to a solution on find_line and possible move to the D-Bus API before I'd do that here.